docs/src/guide/configuration.md

```diff
@@ -26,5 +26,9 @@ in `manifests/metacontroller.yaml`):
 | `--events-qps` | Rate of events flowing per object (default - 1 event per 5 minutes, e.g. `--events-qps=0.0033`) |
 | `--events-burst` | Number of events allowed to send per object (default 25, e.g. `--events-burst=25`) |
 | `--pprof-address` | Enable pprof and bind to endpoint /debug/pprof, set to 0 to disable pprof serving (default 0, e.g. `--pprof-address=:6060`) |
+| `--leader-election` | Determines whether or not to use leader election when starting metacontroller (default `false`, e.g., `--leader-election`) |
+| `--leader-election-resource-lock` | Determines which resource lock to use for leader election (default `leases`, e.g., `--leader-election-resource-lock=configmapsleases`). See the client-go documentation [leaderelection/resourcelock](https://pkg.go.dev/k8s.io/client-go/tools/leaderelection/resourcelock#pkg-constants) for valid values. |
+| `--leader-election-namespace` | Determines the namespace in which the leader election resource will be created (default if in-cluster, current namespace of metacontroller. Otherwise, undefined. e.g., `--leader-election-namespace=metacontroller`) |
```

Comment:
Can this message be improved. I am not able to understand it.

Reply:
I improved the documentation, please let me know if it makes more sense now.